content/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Data Together"
+tagline: "Communities Stewarding Data Together"
+date: 2017-09-10T18:00:00-04:00
+draft: false
+---
+
+<div class="signup container">
+  <div class="row">
+    <div class="text col-md-6 col-sm-12 col-xs-12">
+      <p class="sub-emphasis">Data Together is a new model for distributed, community-driven stewardship of data. This model shows how groups of people, companies, institutions, NGOs, and governments can use decentralized and peer-to-peer web infrastructure to access, discover, verify, and preserve data they care about. We hope to build a future in which networks of collaborators make their data accessible to their peers, immediately discoverable, easily verifiable, and robustly preserved.</p>
+      <!-- <a class="btn btn-large bg-red white" href="/primers">Get Started</a> -->
+    </div>
+    <div class="image col-md-4 offset-md-2"><img src="https://s3.us-east-2.amazonaws.com/static.archivers.space/add-metadata.png"></div>
+  </div>
+</div>
+<div class="left" style="height: 200px; width: 20%; max-width: 200px; margin-right: 25px; margin-top: 100px;">
+  <img class="left-lines" src="https://s3.amazonaws.com/datatogether/svg/lines_left.svg">
+</div>
+<div class="signup dweb-view container">
+  <div class="row">
+    <div class="image col-md-4"></div>
+    <div class="text col-md-6 col-sm-12 col-xs-12">
+      <p class="emphasis">
+        This site is not a centralized data hosting service. It’s a view onto the decentralized network of individuals, organizations, projects and communities who are using the Data Together model.
+      </p>
+    </div>
+  </div>
+</div>
+
+<div class="right" style="height: 200px; width: 20%; max-width: 300px; margin-left: 25px; margin-top: 100px;">
+  <img class="right-lines" src="https://s3.amazonaws.com/datatogether/svg/lines_right.svg">
+</div>
+<div class="signup community container">
+  <div class="row">
+    <div class="text col-md-6 col-sm-12 col-xs-12 sub-emphasis">
+      <p>
+        Data Together is an ongoing, inclusive conversation, an open source community, a collaboration, a model, and a collection of tools, all of which are evolving quickly. To achieve our aims, we are
+        <ul>
+          <li>Developing conceptual and technical models for communities, using decentralized and peer-to-peer infrastructure</li>
+          <li>Articulating the patterns of actors and technologies involved, and value propositions for those actors</li>
+          <li>Creating proof-of-concept tools that communities can use to collectively steward data</li>
+        </ul>
+      </p>
+      <p>
+        There are many ways to get involved and to contribute, the most important of which  is to test the Data Together model with your community and then share back where things work, where the approach opens new possibilities, and where there’s room for improvement.
```

Comment:
The expression "share back" sounds weird. How about "share your experiences about"?